web/static/js/phoenix.js

```diff
@@ -724,7 +724,7 @@ export class Ajax {
       this.xdomainRequest(req, method, endPoint, body, timeout, ontimeout, callback)
     } else {
       let req = window.XMLHttpRequest ?
-                  new XMLHttpRequest() : // IE7+, Firefox, Chrome, Opera, Safari
+                  new window.XMLHttpRequest() : // IE7+, Firefox, Chrome, Opera, Safari
```

Comment:
Because the `jsdom-global` package gives us `window` as a property of `global`, `window` does not represent the global context. That's why I needed to make this change in `phoenix.js` to make sure we can locate the fake `XMLHttpRequest` in our node tests.